### ISSUE TITLE
Piral CLI creation results in build error

### ISSUE THREAD

**Issue by kpyfan** [smapiot/piral#91]
# Bug Report

For more information, see the `CONTRIBUTING` guide.

## Prerequisites

- [x] Can you reproduce the problem in a [MWE](https://en.wikipedia.org/wiki/Minimal_working_example)?
- [x] Are you running the latest version?
- [x] Did you perform a search in the issues?

## Environment Details and Version

Piral CLI Version `0.8.3`

## Description

Creating a new Piral instance from CLI and trying to run it as an example fails with a build issue.

```
/Users/kpyfan/git/piral-poc/src/index.tsx:14:14: Cannot resolve dependency 'piral/api'
    at Resolver.resolve (/Users/kpyfan/git/piral-poc/node_modules/parcel-bundler/src/Resolver.js:71:17)
    at async Bundler.resolveAsset (/Users/kpyfan/git/piral-poc/node_modules/parcel-bundler/src/Bundler.js:433:18)
    at async Bundler.resolveDep (/Users/kpyfan/git/piral-poc/node_modules/parcel-bundler/src/Bundler.js:484:14)
    at async /Users/kpyfan/git/piral-poc/node_modules/parcel-bundler/src/Bundler.js:608:26
    at async Promise.all (index 4)
    at async Bundler.loadAsset (/Users/kpyfan/git/piral-poc/node_modules/parcel-bundler/src/Bundler.js:599:21)
    at async Bundler.processAsset (/Users/kpyfan/git/piral-poc/node_modules/parcel-bundler/src/Bundler.js:557:5)
    at async PromiseQueue._runJob (/Users/kpyfan/git/piral-poc/node_modules/parcel-bundler/src/utils/PromiseQueue.js:48:7)
```

## Steps to Reproduce

1. Initialize a new Piral project using Piral cli (`piral new --target piral-poc`)
2. Run Piral instance using Piral cli (`piral debug`)

## Expected behavior

A working piral instance should be stood up and a sample page should render

## Actual behavior

Parcel Bundler cannot find one of the dependencies

**Comment by FlorianRappl** [comment]
Hi @kpyfan - thanks for raising the issue.

This is indeed a bug in the template for 0.8.3. Even though this may turn off people we've decided against shipping a hotfix for this. As a workaround just change the export to

```
export * from 'piral';
```

Alternatively, go for `@next` (e.g., `piral new --target piral-poc --tag next`). This is the 0.9 preview, which should be already good enough for getting the hands dirty, too.

Sorry for the inconvenience! :beers:

(I'll leave the item open until 0.9 is released)

**Comment by FlorianRappl** [comment]
**Remark**:
For the 0.9 preview to work correctly please also update the tooling to the `@next` preview level. `npm i piral-cli@next`. Thanks!

**Comment by kpyfan** [comment]
Great, I've applied the fix locally and can confirm that it works. Thanks for the quick response!

**Comment by FlorianRappl** [comment]
We've shipped this now already with a 0.8.4 together with 2 more bug fixes / improvements.
